dev/com.ibm.ws.jdbc/src/com/ibm/ws/rsadapter/jdbc/WSJdbcProxyMethod.java

```diff
@@ -211,9 +211,10 @@
         //  oracle.jdbc.internal.OracleConnection.unwrap()
         
         //// PostgreSQL methods
-        unsafeMethods.add("getLargeObjectAPI"); // org.postgresql.PGConnection.getLargeObjectAPI() -- LargeObject API is not allowed because it has operations that can commit transactions on the underlying PG connection
-        unsafeMethods.add("getFastpathAPI"); // org.postgresql.PGConnection.getFastpathAPI() -- this is deprecated anyway on the PG API
-        unsafeMethods.add("setAutosave"); // org.postgresql.PGConnection.setAutosave(AutoSave)
+        // These methods were briefly blocked during 19.0.0.7
```

Comment:
It's not necessary to create source control history using commented out code, remove.